config/crd/bases/actions.summerwind.dev_runners.yaml

```diff
@@ -5160,9 +5195,3 @@ spec:
       subresources:
         status: {}
   preserveUnknownFields: false
-status:
```

Comment:
I don't know why is this removed, but if this is required, it should be generated into model. This should rely on kubebuilder generation, so if this has to be manually added each time, either we should automate it, or extend the model in a way to accept it. Not even sure what this field is. My understanding is that k8s is populating status fields and if CRD needs it, we should add it to the model, and not by hand like this.
Anyway, this works on multiple runners in the same namespace